src/models/i_appmodel.ts

```diff
@@ -1,3 +1,22 @@
+import { ComboboxItem } from "shared/components/Combobox";
```

Comment:
Remove this

Reply:
The same stuff about Type

Reply:
OK, I'll make the change when I simplify the code.